[2024-03-03T03:42:15.089+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_4 scheduled__2016-12-25T00:00:00+00:00 [queued]>
[2024-03-03T03:42:15.102+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_4 scheduled__2016-12-25T00:00:00+00:00 [queued]>
[2024-03-03T03:42:15.104+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:42:15.126+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_4> on 2016-12-25 00:00:00+00:00
[2024-03-03T03:42:15.134+0000] {standard_task_runner.py:60} INFO - Started process 4765 to run task
[2024-03-03T03:42:15.140+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_4', 'scheduled__2016-12-25T00:00:00+00:00', '--job-id', '2176', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmpoc896_k4']
[2024-03-03T03:42:15.145+0000] {standard_task_runner.py:88} INFO - Job 2176: Subtask get_files_4
[2024-03-03T03:42:15.180+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:42:15.257+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_4 scheduled__2016-12-25T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:42:15.394+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_4' AIRFLOW_CTX_EXECUTION_DATE='2016-12-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-12-25T00:00:00+00:00'
[2024-03-03T03:42:17.267+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:42:17.286+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_4, execution_date=20161225T000000, start_date=20240303T034215, end_date=20240303T034217
[2024-03-03T03:42:17.329+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:42:17.363+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
